pkgs/servers/teleport/default.nix

```diff
@@ -19,26 +19,23 @@ buildGoPackage rec {
 
   nativeBuildInputs = [ zip ];
 
-  postBuild = ''
-    pushd .
-    cd $NIX_BUILD_TOP/go/src/github.com/gravitational/teleport
-    mkdir -p build
-    echo "making webassets"
-    make build/webassets.zip
-    cat build/webassets.zip >> $NIX_BUILD_TOP/go/bin/teleport
-    rm -fr build/webassets.zip
-    cd $NIX_BUILD_TOP/go/bin
-    zip -q -A teleport
-    popd
-  '';
+  # Reduce closure size for client machines
+  outputs = [ "out" "client" ];
 
+  buildTargets = [ "full" ];
   dontStrip = true;
```

Comment:
```suggestion
```

Reply:
I am not sure I can strip all binaries, the "teleport" binary contains some assests for the webapp and as the package was using `dontStrip` I wouldn't want to break the teleport server

Reply:
Please test if removing dontStrip is safe.